### ISSUE TITLE
Modify disappear animation direction?

### ISSUE THREAD

**Issue by sonicnkt** [marcinjakubowski/notification-position-reloaded#7]
Im using this extension to show notifications on the bottom right as im using Dash-to-Panel.
In the settings you can modify the animation direction when a notification is created but its not possible to change the direction for when it disappears. 

On my system they always move away to the top of the screen which looks a bit glitchy on my 60hz monitor. 

It would be great if  you could set this manually or if it would default to the reverse direction. (Slide from the bottom - > Slide to the bottom).  An Option to just appear/disappear without animation would be great as well.

**Comment by sonicnkt** [comment]
Weird... 

My tests were running on wayland with a dual monitor setup (both same resolution and orientation) on Gnome 42.1. 
I had my right monitor configured as primary with dash to panel on the bottom. 

I tried it on my personal system now with Gnome 42.1, X11 (NVIDIA).
Here it doesnt work at all... it is also a multimonitor setup but the left display is rotated and smaller than the right one. 
With this setup the notifications always appears on the top left of the right (main) display and the options to change the location don't do anything.

**Comment by marcinjakubowski** [comment]
Haven't used Wayland, but I'm also on Gnome 42.1, X11, NVIDIA and have notifications on my main/right screen, with Dash to Panel on its left edge. Would you go to looking glass (Alt+F2, then type lg and enter) and type the following in the console:

```
Main.layoutManager.getWorkAreaForMonitor(global.display.get_current_monitor())).width
Main.layoutManager.getWorkAreaForMonitor(global.display.get_current_monitor())).height
```

And let me know what it says and how that compares to your resolution?

(You can then leave LG with Esc)

**Comment by sonicnkt** [comment]
I did a small recording of the Issue on my WAYLAND Setup at work:

https://user-images.githubusercontent.com/5454622/169014809-e6d1b31a-8822-4846-969b-d6d1d35c9908.mp4

The output of those lg commands is the following:
```
height = 1398
width = 2560 
```

I will add the info of my X11/Nvidia System later.

**Comment by marcinjakubowski** [comment]
fixed in [590846c](https://github.com/marcinjakubowski/notification-position-reloaded/commit/590846ca38539954e823f2ede3379a84c936804b)
